Network Class/Network.cpp

```diff
@@ -218,6 +218,66 @@ int SGD()
 	return numCorrect;
 }
 
+// thought it might be nice if train also returned a vector of the network's efficiency at each epoch
```

Comment:
Yeah, I think we should create a file with the efficiency at every epoch of the network in train() itself. Because writing an accessor function, and calling train() to store the output in a different vector will call all the other methods again.